hub/powertoys/powerrename.md

```diff
@@ -87,7 +87,27 @@ Choose between four options to either convert items to be all lowercase, all upp
 
 ### Enumerate items
 
-Appends a numeric suffix to file names that were modified in the operation. For example: `foo.jpg` → `foo (1).jpg`
+If selected, you can use the following patterns as part of the _Replace with_ text:
+
+| Variable pattern | Explanation
```

Comment:
The content of this change looks good, but could you add closing pipes `|` to the table (the header row and each item row)? Thanks!

Reply:
done :)